boards/FSM/Inc/main.h

```diff
@@ -87,8 +87,8 @@ extern "C"
     (TIMx_FREQUENCY / TIM17_AUTO_RELOAD_REG / TIM17_PWM_MINIMUM_FREQUENCY)
 #define TIM3_PRESCALER 72
 #define ADC_FREQUENCY 1000
-#define TIM1_AUTO_RELOAD_REG 3599
-#define TIM2_AUTO_RELOAD_REG 1799
+#define TIM1_AUTO_RELOAD_REG 10000
```

Comment:
how were these values chosen?

Reply:
they're equal to 2x full revolution. choice was pretty arbitrary since we never want to auto reload, just big enough so it'll never happen in the car

Reply:
Note: the auto reload register is specifying the maximum value that can be read by the encoder before overflowing (approx 2x bigger than the full revolution of the encoder). This gives us a safe (large) range to detect underflow and reset the encoder